code-challenges.test.js

```diff
@@ -24,11 +43,45 @@
 
 // b) Create the function that makes the test pass.
 
+// 
+// 
+
```

Comment:
Look like you push this branch without your pseudocode explaining your function you used to pass the test